packages/devextreme/scss/widgets/base/_gridBase.scss

```diff
@@ -56,8 +56,11 @@
     }
 
     > .dx-#{$widget-name}-rowsview.dx-last-row-border {
-      tbody:last-child > .dx-data-row:nth-last-child(2) > td {
-        border-bottom-width: 0;
+      tbody:last-child > .dx-data-row:nth-last-child(2) {
+        &,
+        & > td {
+          border-bottom-width: 0;
+        }
```

Comment:
Can we re-write this selector to get rid of the ```&```?
I'm not sure, but it looks like a reversive selector :)

Reply:
Sorry, I was wrong here.
The ```&``` is best solution here :)